01-ruby-fundamentals/exercises/while-worksheet.md

```diff
@@ -0,0 +1,55 @@
+# While Loop Evaluation Worksheet
+For this exercise, you will be evaluating some Ruby code and coming up with scenarios for how the code will execute.
+
+1. For the code below, describe what the loop will do with the following input to the `word` variable. Be sure to include how many times the loop will execute.
+  1. "ham"
+  2. "hamster"
+  3. "bananas"
```

Comment:
They represent 3 separate runs. For the last one, I want them to identify that it will prompt for a new word. I don't want to put multiple words in because I think it gives away the result.